deploy/charts/burrito/templates/runner.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
not sure this is necessary. If the user wants to use `burrito-system` ns as a tenant he can declare in the tenants with `namespace.create: false`

Reply:
Yes I think this is not necessary to have a burrito-runner service account having the same access as the burrito-controllers in the "system" namespace

Reply:
template deleted in a6b078d1751dc3d10f8f342a89eecdfc9c8e8ee0